src/fhir/input/fsh/profiles/LocationDirectory.fsh

```diff
@@ -12,17 +12,23 @@ organisations specific for german Healthcare and Telematics Infrastructure.
 * meta.tag contains Origin 1..1 MS
 * meta.tag[Origin] from OriginVS
 * meta.tag[Origin].system = "https://gematik.de/fhir/directory/CodeSystem/Origin"
+* identifier MS
+* identifier ^slicing.discriminator.type = #value
+* identifier ^slicing.discriminator.path = "system"
+* identifier ^slicing.rules = #open
+* identifier contains BSNR 0..1 MS
+* identifier[BSNR] only $IdentifierBSNR
```

Comment:
Location ist aus meiner Sicht ein Gebäude, in dem sich auch mehrere Praxen/Firmen befinden können. 
Ich würde die BSNR deshalb an eine logische Ressource hängen. So muss ich für jede BSNR mit der gleichen Adresse eine eigene Location erzeugen.

Reply:
der Argumentation kann ich folgen